### PR TITLE
Convert constants to Singleton class

### PR DESCRIPTION
**Pull request by khmikkelsen**
fixed some formalities.
Added new util class ConstantSingleton where all args are passed to.

### CODE REVIEW

File: core/yolo/yolo_func.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from typing import Tuple
 from ultralytics import YOLO
+from PIL import Image
 
+Image.MAX_IMAGE_PIXELS = None
 
-def obj_det(img, model: YOLO, conf=0.25):
-    res = model.predict(source=img, save=False, save_txt=False, imgsz=640, conf=conf, device="cpu")
+
+def obj_det(img, model: YOLO, conf=0.25, img_size=(640, 640)):
```
Comment (lucas2000k) [inline core/yolo/yolo_func.py:8]: hvorfor giver vi en tuple som img_size her nu?

Comment (khmikkelsen) [reply core/yolo/yolo_func.py:8]: Tried forcing yolo to return a square image. Works 'sometimes'.

Comment (lucas2000k) [reply core/yolo/yolo_func.py:8]: but why though?

Comment (khmikkelsen) [reply core/yolo/yolo_func.py:8]: black bars = bad. check image in #101

Comment (khmikkelsen) [reply core/yolo/yolo_func.py]: fixed in latets

**Comment by khmikkelsen** [comment]
updated for easy review :+1: @lucas2000k

**Review by lucas2000k — APPROVED** [review]
Looks good

**Pull request merged**